eox_tagging/test/test_viewset.py

```diff
@@ -63,18 +63,37 @@ def test_retreive_tag(self):
 
     def test_create_tag(self):
```

Comment:
This test verifies the response with `"owner_type": "user",` could you add a test with` "owner_type": "site"` ? and could you verify that the owner_type is not None, since the tests show that the line https://github.com/eduNEXT/eox-tagging/blob/72fde4da1a29ea5c8780039192beb2e5ea408702/eox_tagging/api/v1/viewset.py#L58 is not been executed

Reply:
Yes! Of course. Let me add those tests

Reply:
The owner_type refers to this: /eox_tagging_api/v1/tags/?owner_type=site or /eox_tagging_api/v1/tags/?owner_type=user. If owner_type is none the queryset returned is an union of both. I'll add tests for that.

Reply:
Done!